ironfish/src/workerPool/tasks/createMinersFee.ts

```diff
@@ -21,23 +21,23 @@ export class CreateMinersFeeRequest extends WorkerMessage {
 
   serialize(): Buffer {
     const bw = bufio.write(this.getSize())
-    bw.writeVarBytes(BigIntUtils.toBytes(this.amount))
+    bw.writeVarBytes(BigIntUtils.toBytesBE(this.amount))
```

Comment:
why are we using BE for this?

Reply:
This is just changing the name of `toBytes` to `toBytesBE` to be more explicit. Not changing any functionality. Although not really sure why we were using BE over LE here although seems somewhat arbitrary since it's just serializing a worker pool message